.travis.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       script:
         - make test
     - <<: *simple-test
-      go: 1.9.x
+      go: 1.11.x
```

Comment:
+2 here, since I think it was missed last time. Was 9, 11, 12, tip - now is 11, 12, 13, tip.